cdap-app-templates/cdap-etl/hydrator-spark-core-base/src/main/java/io/cdap/cdap/etl/spark/batch/BatchSparkPipelineDriver.java

```diff
@@ -191,9 +191,12 @@ public void run(DatasetContext context) throws Exception {
         sec.getRuntimeArguments().getOrDefault(Constants.CONSOLIDATE_STAGES, Boolean.TRUE.toString()));
       boolean shouldCacheFunctions = Boolean.parseBoolean(
         sec.getRuntimeArguments().getOrDefault(Constants.CACHE_FUNCTIONS, Boolean.TRUE.toString()));
+      boolean isPreviewEnabled =
```

Comment:
nit: it's not preview "is enabled", it is if it's a preview run or regular run, correct?

Reply:
We already use the `isPreviewEnabled` semantics elsewhere in the app, I wanted to keep it consistent for familiarity.